File: app/src/main/java/teamenglify/englify/Model/VocabPart.java
package teamenglify.englify.Model;

import java.io.Serializable;
import java.util.Date;

/**
 * Created by Keith on 07-Mar-17.
 */

public class VocabPart implements Serializable {
    public String text;
    public String imgURL;
    public String audioURL;
    public Date lastModified;

    public VocabPart(String text, String imgURL, String audioURL) {
        this.text = text;
        this.imgURL = imgURL;
        this.audioURL = audioURL;
    }

    public VocabPart(String text, String imgURL, String audioURL, Date lastModified) {
        this.text = text;
        this.imgURL = imgURL;
        this.audioURL = audioURL;
        this.lastModified = lastModified;
    }
}

File: app/src/main/java/teamenglify/englify/Model/realm/VocabPart.kt
package teamenglify.englify.Model.realm

import io.realm.RealmObject

open class VocabPart: RealmObject() {
    //From firebase real-time store
    lateinit var id: String
    lateinit var text: String
    var translation: String? = null
    var imageUrl: String? = null
    var audioUrl: String? = null

    //From firebase storage
    var imageData: ByteArray? = null
    var audioData: ByteArray? = null
}